tests/fixtures/golangci-lint/golangci-fixer.yml

```diff
@@ -0,0 +1,4 @@
+---
+linters:
+  enable:
+    - gofmt
```

Comment:
The golangci-lint tool has been removed. I never got it working on real world repositories. It would consume 2GB+ of ram and then be killed by OOMkiller.

Reply:
Oops sorry, I re-implemented it for our system, but forgot to delete the fixtures from this PR